package/src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -70,6 +70,19 @@ const parse: ParseFunction = (capture, parse, state) => ({
   content: parseInline(parse, capture[0], state),
 });
 
+const getUrlsFromText = (text: string) => {
```

Comment:
I like wrapping the dependency in our own function :) `parseUrlsFromText` might be a tiny bit clearer on what the aim of the function is, but not important either way. 👍🏻  :)